concur/widgets.py

```diff
@@ -561,18 +561,18 @@ def drag_drop_target(tag, value, widget):
         yield
 
 
-def columns(elems, identifier=None, vert_borders=True, hor_borders=True, widths=None) -> Widget:
+def columns(elems, identifier=None, vert_borders=True, hor_borders=True, widths=None):
     """
     Table, using the imgui columns API.
-    :param List[List[Widget]] elems: 2D array of widgets. A list containing another list for
-    each row. len() of the first row defines the amount of columns, that will be rendered.
-    :param identifier: TODO
-    :param bool hor_borders: Toggle horizontal borders on/off
-    :param List[Union[int,float]] widths: is a optional vector of column widths in pixels. May
-    contain  None values.
-    :param bool vert_borders: Toggle horizontal borders on/off
-    :return: orr() wrapping the column widget
-    :rtype: WidgetWrapper
+    Args:
+        elems (List[List[Widget]]): 2D array of widgets. A list containing another list for each row. len() of the first row defines the amount of columns, that will be rendered.
```

Comment:
This seems to imply that rows can be of different lengths, which is not the case. Could you add the fact that rows must have the same length to the docstring?

Reply:
Good point, yes.